examples/text_summarization/summarization_evaluation.ipynb

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
<p>Does compute_rouge_perl support other natural languages? If so, can you provide a similar Hindi example; if not, can you mention it in the writing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp-recipes/pull/526/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
No, compute_rouge_perl doesn't support other languages. It's going to be hard to hack it to do it since I don't know perl. I mentioned it in the introduction section.